src/lib/Libpython/shared_python_utils.c

```diff
@@ -43,28 +43,62 @@
 #include "pbs_internal.h"
 #include "log.h"
 
+
+#ifdef WIN32
+
+/**
+ * @brief get_py_homepath
+ * 	Find and return where python home is located
+ *
+ * @param[in] homepath - buffer to copy python home path
+ *
+ * @return int
+ * @retval 0 - Success
+ * @retval 1 - Fail
+ */
+
+int
+get_py_homepath(char **homepath) {
+#ifdef PYTHON
+	static char python_homepath[MAXPATHLEN + 1] = {'\0'};
+	if (python_homepath[0] == '\0') {
+		snprintf(python_homepath, MAXPATHLEN, "%spython", pbs_conf.pbs_exec_path);
+		forward2back_slash(python_homepath);
+		if (!file_exists(python_homepath)) {
+				log_err(-1, __func__, "Python home not found!");
+				return 1;
+		}
+	}
+	*homepath = strdup(python_homepath);
+	if (*homepath == NULL)
+		return 1;
+	return 0;
+#else
+	return 1;
+#endif
+}
+#endif
+
 /**
  * @brief get_py_progname
  * 	Find and return where python binary is located
  *
- * @param[in] dest - buffer to copy python path
- * @param[in] dest_sz - size of dest
+ * @param[in] binpath - buffer to copy python binary path
```

Comment:
Add to doxygen that the caller is responsible for free()ing the memory allocated